src/selectors/getContextsSortedAndRanked.ts

```diff
@@ -28,12 +29,13 @@ const getContextsSortedAndRanked = (state: State, value: string): Thought[] => {
         ...(thought || {
           id: cxid,
           childrenMap: {},
-          parentId: '', // ???
+          parentId: '' as ThoughtId, // ???
```

Comment:
I should've not touched this maybe, but 
`thought` comes from `getThoughtById` and the types don't match: we need `created` and `parentId: ThoughtId` as a "replacement" for our "missing" thought, so I thought I gotta fix this TS issue

Reply:
That's fine. Clearly `''` is not a valid `ThoughtId`, but it is being treated as one already, so there is no harm in adding the type assertion.